src/libs/actions/Policy.js

```diff
@@ -91,6 +91,54 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Remove the passed members from the policy employeeList
+ *
+ * @param {Array} members
+ * @param {String} policyID
+ */
+function removeMembers(members, policyID) {
+
+    // Check if the members array is not empty
```

Comment:
It's pretty obvious what the check is doing. I'd suggest we do one of these:

1. Improve this comment so it explains "why" and not "what"
2. Remove this comment since it should be obvious to anyone looking at this why we would return early in this case.

Reply:
I chose to update the comment as this is a consequence of deciding to filter out the admin email from member list in different file, therefore something which might not be clear to other developers when inspecting the code at first glance.